site/src/docs/components/header/index.mdx

```diff
@@ -22,56 +22,232 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
+    </Header.ActionBar>
+    <Header.NavigationMenu>
+      <Header.NavigationLink
+        href="#"
+        label="Link 1"
+        onClick={(event) => event.preventDefault()}
+        active
+        dropdownLinks={[
+          <Header.NavigationLink
+            href="#"
+            label="Nested link 1"
+            active
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Deep link 1" />,
+              <Header.NavigationLink href="#" label="Deep link 2" />,
+              <Header.NavigationLink href="#" label="Deep link 3" />,
+            ]}
+          />,
+          <Header.NavigationLink
+            href="#"
+            label="Nested link 2"
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Deep link 1" />,
+              <Header.NavigationLink href="#" label="Deep link 2" />,
+            ]}
+          />,
+        ]}
+      />
+      <Header.NavigationLink
+        href="#"
+        label="Link 2"
+        dropdownLinks={[
+          <Header.NavigationLink
+            href="#"
+            label="Nested link 1"
+            active
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Deep link 1" />,
+              <Header.NavigationLink href="#" label="Deep link 2" />,
+            ]}
+          />,
+          <Header.NavigationLink
+            href="#"
+            label="Nested link 2"
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Deep link 1" />,
+              <Header.NavigationLink href="#" label="Deep link 2" />,
+            ]}
+          />,
+        ]}
+      />
+      <Header.NavigationLink href="#" label="Link 3" />
+    </Header.NavigationMenu>
+  </Header>
+</PlaygroundPreview>
 
-#### With navigation bar
+#### With universal bar
 
-Coming soon!
+- Using the Header.UniversalBar component is optional. Use it to provide frequently visited and useful links.
+- If the primary link information is provided, it should direct to the webpage of City of Helsinki.
+- Keep the link labels clear and concise. Prefer max 1-2 word long labels so the labels don't go to the next line on smaller screens. Avoid starting multiple link labels with the same word.
+- On smaller screens (768px and down) the Header.UniversalBar is hidden and the contents are moved inside the [the Header.ActionBar component's](#with-action-bar) menu.
+
+<PlaygroundPreview>
+  <Header>
+    <Header.UniversalBar primaryLinkHref="#" primaryLinkText="Helsingin kaupunki">
+      <Header.NavigationLink href="#" label="F.A.Q" />
+      <Header.NavigationLink href="#" label="User guide" />
+    </Header.UniversalBar>
+    <Header.ActionBar
+      title="Helsingin kaupunki"
+      titleHref="https://hel.fi"
+      titleHref="https://hel.fi"
+      logoAriaLabel="Service logo"
+      logoHref="https://hel.fi"
+      menuButtonAriaLabel="Menu"
+    />
+  </Header>
+</PlaygroundPreview>
 
 #### With navigation language selector
 
```

Comment:
Language selector changes the language for the whole service. It is recommended to offer full Finnish, Swedish and English versions. Other languages can be added based on the need and target audience of the service. Smaller subset of content can be separated from fully translated content.